README.md

```diff
@@ -1,6 +1,6 @@
 # Jekyll Title from Headings
 
-*A Jekyll plugin to pull page the title from fist Markdown heading when none is specified.*
```

Comment:
As pointed out in #9, there's another error in this sentence.
You would want it to be "pull the page title" rather than "pull page the title"